interfaces/devdoc/threadpool_requirements.md

```diff
@@ -60,35 +62,45 @@ MOCKABLE_FUNCTION(, THREADPOOL_HANDLE, threadpool_create, EXECUTION_ENGINE_HANDL
 
 **NON_THREADPOOL_01_002: [** If `execution_engine` is `NULL`, `threadpool_create` shall fail and return `NULL`. **]**
 
+[Will be moved from threadpool_open.]
+**N_O_N_THREADPOOL_01_011: [** `threadpool_open` shall switch the state to OPENING and perform any actions needed to open the `threadpool` object. **]**
```

Comment:
>OPENING [](http://example.com/codeflow?start=75&length=7)

I don't think we have an OPENING state, we said we are removing that.

Reply:
Yes. But I thought it might affect state transitioning signals that API users might be using in their logic or tests which may fail. I know this would be fast transitioning between OPENING and then OPEN, but I just felt like some tests may fail somewhere.

Reply:
Corrected after discussing offline.